### PR TITLE
Feature/use streamid in stream mds

### PR DESCRIPTION
**Pull request by anwar6953**
Update Stream MetaStore to utilize Id.Stream.

Build: http://builds.cask.co/browse/CDAP-DUT775-2

### CODE REVIEW

File: cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamFetchHandler.java [hunk 1]
```diff
@@ -107,11 +107,11 @@ public void fetch(HttpRequest request, HttpResponder responder,
                     @QueryParam("limit") @DefaultValue("2147483647") int limit) throws Exception {
 
     String accountID = getAuthenticatedAccountId(request);
-    if (!verifyGetEventsRequest(accountID, stream, startTime, endTime, limit, responder)) {
+    Id.Stream streamId = Id.Stream.from(accountID, stream);
```
Comment (bdmogal) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamFetchHandler.java]: nit: namespaceId

Comment (bdmogal) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamFetchHandler.java]: Ok. Spoke offline. Ignore for now. more changes coming.

File: cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java [hunk 2]
```diff
@@ -68,7 +68,7 @@ public LocalStreamService(StreamCoordinatorClient streamCoordinatorClient,
 
   @Override
   protected void initialize() throws Exception {
-    String namespace = Constants.DEFAULT_NAMESPACE;
+    Id.Namespace namespace = Id.Namespace.from(Constants.DEFAULT_NAMESPACE);
```
Comment (bdmogal) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java]: Presume this will change soon?

**Comment by bdmogal** [comment]
LGTM

File: cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java (hunk 2 above)
Comment (anwar6953) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java]: yea, thats a WIP at:
feature/CDAP-1314-namespaced-streams

**Pull request merged**